charter.html

```diff
@@ -241,6 +241,17 @@ <h2 id=proposals>Proposals</h2>
 expressed interest in the Proposal
 </ul>
 
+<p>Proposals result in
```

Comment:
They may result in, or *be* explainers in many cases, right? 

For example Microsoft will likely prepare an explainer first and submit it as a Proposal to discuss. We host these in one canonical location on our organization's GitHub initially, and then link folks to them for comment. If it looks like an explainer will have traction, then we "adopt" (copy) it into it's new home for future development, retaining an archived copy on the original site. (That's just our process.)

Reply:
Fixed in 32f4c98.